planet/cli/data.py

```diff
@@ -589,18 +589,19 @@ async def asset_wait(ctx,
                      max_attempts):
     '''Wait for an asset to be activated.
 
-    Returns when the asset state has reached "activated" and the asset is
+    Returns when the asset status has reached "activated" and the asset is
     available.
     '''
     quiet = ctx.obj['QUIET']
     async with data_client(ctx) as cl:
         asset = await cl.get_asset(item_type.pop(), item_id, asset_type_id)
-        with StateBar(order_id="my asset", disable=quiet) as bar:
-            state = await cl.wait_asset(asset,
-                                        delay,
-                                        max_attempts,
-                                        callback=bar.update_state)
-    click.echo(state)
+        with AssetStatusBar(item_type, item_id, asset_type_id,
```

Comment:
I think the issue with `item_type` being reported as an empty list comes from `item_type` being brought in as a `CommaSeparatedString`. 

When I change `item_type` to a string I get:
`00:00 - PSScene 20221003_002705_38_2461 basic_udm2 - status: active`

If you merge in recent changes from your base branch, this should fix this!